libraries/python.md

```diff
@@ -50,23 +50,34 @@ aws_secret_access_key = {SERVICE_INSTANCE_ID}
 
 If both `~/.bluemix/cos_credentials` and `~/.aws/credentials` exist, `cos_credentials` will take preference.
 
+### Gather required information
+
+The following variables appear in the examples:
+
+* `bucket_name` must be a  [unique and DNS-safe](docs/services/cloud-object-storage/api-reference/api-reference-buckets.html#compatibility-api-new-bucket) string. Because bucket names are unique, these values will need to be changed if this example is run multiple times.  Note that names are reserved for 10-15 minutes after deletion.
```

Comment:
We should note unique and DNS-safe system wide. Some people assume its for their account or bucket, and that is not the case.

Reply:
Great point.